be/src/column/map_column.cpp

```diff
@@ -621,4 +621,35 @@ Status MapColumn::unfold_const_children(const starrocks::TypeDescriptor& type) {
     return Status::OK();
 }
 
+// keep the last identical key
+void MapColumn::remove_duplicated_keys() {
```

Comment:
the first way break the vectorization execution of expressions.
the second way need copy following un-related maps.
we should refactor the map to avoid the second problem in the future. For example adding an size column for each map besides offsets, so that we can move map items within a map, no need to moving following un-changed maps.